### PR TITLE
[DO NOT MERGE] PR for discussion of https://redmine.deriv.cloud/issues/6019#logs/standardise_logging

### PR DESCRIPTION
**Pull request by shantanu-deriv**
@michaelmueller-binary I am creating this fork so that we can review changes easily. This will be an emtpy file so that the diff is entire changes of your original repo

### CODE REVIEW

File: lib/Log/Any/Adapter/JSON.pm [hunk 1]
```diff
@@ -1,199 +0,0 @@
...
-    }
-    return (\@stack);
-}
-
-# These loops create the methods that Log::Any expects to be present for each 
-# log level. 
-foreach my $method (Log::Any::Adapter::Util::logging_methods()) {
-    no strict 'refs';
-    my $method_level = Log::Any::Adapter::Util::numeric_level($method);
-    *{$method} = sub { $_[0]->log($_[1]) }
```
Comment (shantanu-deriv) [inline lib/Log/Any/Adapter/JSON.pm:187]: can we set `$_[0]` and `$_[1]` to meaningful variable names

File: lib/Log/Any/Adapter/JSON.pm [hunk 2]
```diff
@@ -1,199 +0,0 @@
...
-}
-
-foreach my $method (Log::Any::Adapter::Util::detection_methods()) {
-    no strict 'refs';
-    my $base         = substr($method, 3);
-    my $method_level = Log::Any::Adapter::Util::numeric_level($base);
-    *{$method} = sub {
-        return !!($method_level <= $_[0]->{log_level});
```
Comment (shantanu-deriv) [inline lib/Log/Any/Adapter/JSON.pm:195]: Same

File: lib/Log/Any/Adapter/JSON.pm [hunk 3]
```diff
@@ -1,199 +0,0 @@
...
-         "package":"main"
-      }, 
-      ...
- ]
-   
-
-=cut
-
-
-sub stack_build {
```
Comment (shantanu-deriv) [inline lib/Log/Any/Adapter/JSON.pm:169]: Is there a way to also return stack trace?

**Comment by michaelmueller-binary** [comment]
Thanks @shantanu-deriv , Note that after discussions with TOM I wont be proceeding with this.

**Comment by shantanu-deriv** [comment]
Noted @michaelmueller-binary

**Pull request closed without merging**